indexer/packages/postgres/src/db/migrations/migration_files/20240913142157_change_affiliate_info_decimal_precision_and_add_total_referred_volume.ts

```diff
@@ -1,22 +0,0 @@
-import * as Knex from 'knex';
-
-export async function up(knex: Knex): Promise<void> {
-  return knex.schema.alterTable('affiliate_info', (table) => {
-    // null indicates variable precision whereas not specifying will result in 8,2 precision,scale
-    table.decimal('affiliateEarnings', null).alter();
-    table.decimal('totalReferredFees', null).alter();
-    table.decimal('referredNetProtocolEarnings', null).alter();
-
-    table.decimal('referredTotalVolume', null).notNullable();
-  });
-}
-
-export async function down(knex: Knex): Promise<void> {
-  return knex.schema.alterTable('affiliate_info', (table) => {
-    table.decimal('affiliateEarnings').alter();
-    table.decimal('totalReferredFees').alter();
-    table.decimal('referredNetProtocolEarnings').alter();
-
-    table.dropColumn('referredTotalVolume');
-  });
-}
```

Comment:
Would deleting a migration cause issues with the database here?

Reply:
The newly added migration and the deleted one are almost the same, with compatible up/downs. For better comptibility, let me modify the file in place instead of adding/deleting.

Reply:
deleting, and modifying migrations are both breaking changes, its always preferred to add a new migration